Benchmark workflows failing due to multiple jars in benchmark distribution

### Describe the bug

The scheduled run is failing due to #996:

https://github.com/keycloak/keycloak-benchmark/actions/runs/17404019861

The root cause of the issue is that the gatling benchmark is failing due to the classpath being incorrectly configured, we can see from the Ansible logs that execution is failing with the following:

```json
"stderr": "java: no process found\nError: Could not find or load main class .opt.keycloak-benchmark-999.0.0-SNAPSHOT.bin....lib.keycloak-benchmark-dataset-999.0.0-SNAPSHOT.jar\nCaused by: java.lang.ClassNotFoundException: /opt/keycloak-benchmark-999/0/0-SNAPSHOT/bin////lib/keycloak-benchmark-dataset-999/0/0-SNAPSHOT/jar\njq: error: Could not open file /opt/keycloak-benchmark-999.0.0-SNAPSHOT/bin/../results///js/stats.json: No such file or directory",
"stderr_lines": [
            "java: no process found",
            "Error: Could not find or load main class .opt.keycloak-benchmark-999.0.0-SNAPSHOT.bin....lib.keycloak-benchmark-dataset-999.0.0-SNAPSHOT.jar",
            "Caused by: java.lang.ClassNotFoundException: /opt/keycloak-benchmark-999/0/0-SNAPSHOT/bin////lib/keycloak-benchmark-dataset-999/0/0-SNAPSHOT/jar",
            "jq: error: Could not open file /opt/keycloak-benchmark-999.0.0-SNAPSHOT/bin/../results///js/stats.json: No such file or directory"
],
```

### Version

Latest

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

--- a/benchmark/src/main/content/bin/kcb.sh
+++ b/benchmark/src/main/content/bin/kcb.sh
@@ -110,7 +110,7 @@ if [ "$DEBUG_MODE" = "true" ]; then
     fi
 fi
 
-CLASSPATH_OPTS="$DIRNAME/../lib/*"
+CLASSPATH_OPTS=$(find $DIRNAME/../lib -type f | paste -sd:)
 
 declare -A RESULT_CACHE
 

